example_client_test.go

```diff
@@ -36,28 +36,32 @@ func Example() {
 
 // Queue represents a connection to a specific queue.
```

Comment:
I know it's not in the original scope but this badly needs renaming. Clients do not connect to queues. `ConsumerState` would be a better name here.

Reply:
What do you think of `Session`?

Reply:
`Session` is pretty good.